plutus-example/src/PlutusExample/ScriptContextChecker.hs

```diff
@@ -247,3 +254,11 @@ dummyPOSIXTimeRange = Plutus.from $ Plutus.POSIXTime 42
 dummyScriptPurpose :: Maybe Plutus.ScriptPurpose
 dummyScriptPurpose = Nothing
 
+getTxInInfoFromTxIn
+  :: Shelley.UTxO (Alonzo.AlonzoEra StandardCrypto)
+  -> Ledger.TxIn StandardCrypto
+  -> Maybe Plutus.TxInInfo
+getTxInInfoFromTxIn (Shelley.UTxO utxoMap) txIn = do
+  txOut <- Map.lookup txIn utxoMap
+  Alonzo.txInfoIn txIn txOut
```

Comment:
@Jimbo4350 Not sure if this is the best approach to get the `TxInInfo` now, since the signature of `Alonzo.txInfoIn` changed. Can you validate?